src/Tenderly.ts

```diff
@@ -32,6 +28,15 @@ export class Tenderly {
   }
 
   public async verify(...contracts) {
+    const priv = this.env.config.tenderly.privateVerification;
+    if (priv !== undefined && priv && this.env.network.name !== "tenderly") {
+      return this.push(...contracts);
+    }
```

Comment:
What's the point of this? The way it worked until now was push = private, verify = public. Do we want to remove push if we're going with private/public aware verify?

Reply:
No, it's pretty much the same as before, just in case `privateVerification` flag is provided we want to be sure we always to private verification and vice-versa.